garak/resources/gcg/attack_manager.py

```diff
@@ -1790,7 +1792,7 @@ def __init__(self, model_name, conv_template):
     def run(tasks, results):
         while True:
             task = tasks.get()
-            if task is None:
+            if not task:
```

Comment:
[pep8](https://peps.python.org/pep-0008/#programming-recommendations) prefers `is` for `NoneType` value checks (which i learned from looking at your code), how come the change?

Reply:
Huh, it looks nicer to me this way, but I can do `is None`!

Reply:
`if not task` looks nicer to me too, just curious if there was maybe another rule